NodeManager.cpp

```diff
@@ -1318,6 +1399,113 @@ void SensorBH1750::onReceive(const MyMessage & message) {
 #endif
 
 /*
+   SensorTSL2561
+*/
+#if MODULE_TSL2561 == 1
+// contructor
+SensorTSL2561::SensorTSL2561(int child_id, TSL2561* tsl, int tsl_gain, int tsl_timing, int tsl_spectrum): Sensor(child_id,A5) {
+  setPresentation(S_LIGHT_LEVEL);
+  setType(V_LEVEL);
+  _tsl_gain = tsl_gain;
+  _tsl_timing = tsl_timing;
+  _tsl_spectrum = tsl_spectrum;
+}
+
+void SensorTSL2561::setGain(int value) {
+  _tsl_gain = value;
+}
+
+void SensorTSL2561::setTiming(int value) {
+  _tsl_timing = value;
+}
+  
+void SensorTSL2561::setSpectrum(int value) {
+  _tsl_spectrum = value;
+}
+
+// what do to during before
+void SensorTSL2561::onBefore() {
+   _tsl->begin();      
+}
+
+// what do to during setup
+void SensorTSL2561::onSetup() {
+  // You can change the gain on the fly, to adapt to brighter/dimmer light situations     
+  switch (_tsl_gain) {
+    case 0:
```

Comment:
Here and below is where you can you the const static int defined in NodeManager.h as mentioned